samples/java-spring-shopping-cart-quickstart/README.md

```diff
@@ -0,0 +1,83 @@
+# Quickstart project: Shopping Cart
+
+## Designing
+
+To understand the Kalix concepts that are the basis for this example, see [Designing services](https://docs.kalix.io/services/development-process.html) in the documentation.
+
+
+## Developing
+
+This project demonstrates the use of an Event Sourced Entity.
+To understand more about these components, see [Developing services](https://docs.kalix.io/services/)
+and in particular the [Java SDK section](https://docs.kalix.io/java/)
+
+## Building
+
+You can use Maven to build your project, which will also take care of
+generating code based on the `.proto` definitions:
```

Comment:
gRPC leftovers on the generated template maybe?

Reply:
Hmm, I don't remember how I created it. Maybe this is something in the template, or I copied from a protobuf sample. I will check.